### ISSUE TITLE
subscription active but doesn't find

### ISSUE THREAD

**Issue by RobertBrunhage** [delightfulgoods/flutter_payments#7]
I have an active 7 day subscription just for testing and I have payed (it's active). The problem now is that I am on the second week and now it says that activePurchase is null so it will display the button instead of the text. If I press the button it says I already have that subscription.

Any idea why that is?

**Comment by RobertBrunhage** [comment]
Also tried the following code:

    List<Purchase> purchaseHistory = await 
        FlutterPayments.getPurchaseHistory(ProductType.Subscription);
      print('purchaseHistory: ${purchaseHistory[0].purchaseExpires}');

And this printed the date the second one should have started but shouldn't it print when it will renew?
When I print isActive it says false etc.

**Comment by Sun3** [comment]
Did you use the ``` SubscriptionManager.instance.initiatePurchase(p);``` to start the Subscription purchase? When I do that it does not do anything. Mine is a Auto Renewable Subscription in iOS. Have not tested Android yet.

Thanks

**Comment by RobertBrunhage** [comment]
Yeah I used that function but only done it in android so far. Will try out more on IOS as soon as I get this fixed or if I can't do anything about it, and @pplante fixes it :).

**Comment by Sun3** [comment]
Thanks for the info. I am going to try on Android next but iOS no go so far.
